README.md

```diff
@@ -87,6 +87,32 @@ export JAVA_HOME=$(/usr/libexec/java_home -v 13)
 - The `ANDROID_HOME` environment variable must be defined (typically %LOCALAPPDATA%/Android/sdk).
 - The `JAVA_HOME` environment variable must be defined (typically %ProgramFiles%/Android/Android Studio/jre).
 
+### **Configuring a Linux Dev Environment**
+
+**Required Tools:** [Android Studio](https://developer.android.com/studio/) (including NDK 21.3.6528147)
+
+With Ubuntu, you can install needed packages by this command:
+
+```
+sudo apt-get install adb ninja-build openjdk-14-jdk android-sdk
+```
+
+Update PATH with this commands:
```

Comment:
Can this be put into some file like on MacOS so it doesn't have to be done every time you open a terminal?

Reply:
You mean for apt-get ? they are system packages you only have to install once. And already installed packages won't be re-downloaded.

Reply:
For the path, you can add that to .bashrc or somewhere else depending on your distro/shell/tastes/...
https://unix.stackexchange.com/questions/26047/how-to-correctly-add-a-path-to-path